scripts/gpu/setup_nvidiadevice.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Follow the Nvidia guide, use helm to install it
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
+   && chmod 700 get_helm.sh \
+   && ./get_helm.sh
+
+helm repo add nvdp https://nvidia.github.io/k8s-device-plugin \
+   && helm repo update
```

Comment:
please move this code to setup_system.sh, making `helm` one of the vHive dependencies.

Reply:
Do we need specify whether NVIDIA is needed for that? As helm is only used for NVIDIA